rtbot_plugin_manager/tests/test_plugin_manager.py

```diff
@@ -180,9 +180,9 @@ def test_load_and_unload_plugin_with_references(pm):
 
     # Here comes the real test: now try to load the module directly: if it's still known to the interpreter ('cached'),
     # it will just load it anyway. If not, it will try to find a file and fail.
-    with pytest.raises(ModuleNotFoundError):
+    # with pytest.raises(ModuleNotFoundError):
```

Comment:
What is the reason for these lines being commented out?

If this was intentional and you had a good reason for it, then you should document that as comments in the code.